src/js/common/sentry.js

```diff
@@ -0,0 +1,22 @@
+import Raven from 'raven-js';
+
+import environment from './helpers/environment';
+
+// url check is necessary for e2e tests
+const trackErrors = (__BUILDTYPE__ === 'production' || __BUILDTYPE__ === 'staging')
```

Comment:
This looks right in combination with the BASE_URL check. Any idea on how best to pull this out into a configuration file or set it up as a feature flag or option?